src/test/java/gov/usgs/wma/waterdata/RedirectControllerTest.java

```diff
@@ -5,21 +5,33 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
 import org.springframework.boot.test.web.client.TestRestTemplate;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 
-@SpringBootTest(webEnvironment=WebEnvironment.RANDOM_PORT)
-public class VersionControllerTest {
+@SpringBootTest(webEnvironment=WebEnvironment.RANDOM_PORT,
```

Comment:
This and related changes allow the getVersionTest to work locally and remotely with no interference from locally defined application.yml files.